src/app/legacy/components/Footer/index.test.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import Footer from '#components/Footer';
+import { render } from '../../../components/react-testing-library-with-providers';
```

Comment:
small one: could we import `screen` here too?

Reply:
Good spot! I have done this and refactored a few other test files in this PR (and also a few legacy/components from my last PR - JIRA ticket 328)